actors/builtin/miner/miner_test.go

```diff
@@ -1693,6 +1710,26 @@ func TestWithdrawBalance(t *testing.T) {
 	})
 }
 
+func TestChangePeerID(t *testing.T) {
+	periodOffset := abi.ChainEpoch(100)
+
+	setupFunc := func() (*mock.Runtime, *actorHarness) {
+		actor := newHarness(t, periodOffset)
```

Comment:
I don't like these setup funcs generally (because they couple different cases and are easy to abuse). But another reason is that this captures the wrong `t`. It must be a parameter to `setupFunc`.